examples/starwars/lib/reviews/review_page_list.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/material.dart';
 import 'package:graphql_flutter/graphql_flutter.dart';
 import 'package:starwars_app/reviews/review_subscription.dart'
     show DisplayReviews;
+import 'package:gql/language.dart';
```

Comment:
I know apollo uses an analogous api with `graphql-tag`, but I find this api pretty non-ergonomic. Can we just re-export `parseString` as `gql` from `graphql/client` and `graphql_flutter`?

Reply:
so instead of parseString(query) we have something like gql(query)?

Reply:
@micimize I think, that’s a great good idea!

Reply:
right - the analogous idiom in apollo is the ``` gql`query {}` ``` template tag

Reply:
Let me see what I can do.

Reply:
it would have been amazing if dart supported [JavaScripts Tagged Template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) Literals and we would have done it like apollo does `gql"query {}"` for now I guess it will have a function.